pkg/orchestrator/etcd_worker.go

```diff
@@ -42,6 +43,11 @@ const (
 	// takes more than etcdWorkerLogsWarnDuration, it will print a log
 	etcdWorkerLogsWarnDuration = 1 * time.Second
 	deletionCounterKey         = "/meta/ticdc-delete-etcd-key-count"
+
+	// ProcessorRole is the role of the processor etcd worker
```

Comment:
We already have these values. See: `pkg/util/identify`.

/hold

Reply:
fixed